release-calendar/package.json

```diff
@@ -24,8 +24,6 @@
     "setup": "npm run setup-build && npm run setup-serve",
     "setup-build": "webpack --config ./webpack.setup.config.js",
     "setup-serve": "node -r dotenv/config ./dist/bundle-setup.js",
-    "setup-build": "webpack --config webpack.setup.config.js",
-    "setup-serve": "node -r dotenv/config dist/bundle-setup.js",
```

Comment:
Don't remove this please

Reply:
👍I thought I had accidentally duplicated. oop

Reply:
Oh weird, it is duplicated. Maybe merge master?